perseo-main/src/main/java/com/telefonica/iot/perseo/EventsServlet.java

```diff
@@ -76,25 +77,25 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
         Utils.putCorrelatorAndTrans(request);
         logger.debug("events doPost");
         response.setCharacterEncoding("UTF-8");
-        response.setContentType("application/json;charset=UTF-8");
-        PrintWriter out = response.getWriter();
-        try {
-            StringBuilder sb = new StringBuilder();
+        response.setContentType("application/json;charset=UTF-8");        
```

Comment:
Wrong indent... again :)

You can self-discover this kind of problems along all the modified files if you have a carefull look to https://github.com/telefonicaid/perseo-core/pull/151/files. Please, do it.

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823